### PR TITLE
60 add tickets feature that allows to add new movie

### PR DESCRIPTION
**Pull request by adlbrtbsr**
- users start with 1 ticket
- adding a movie costs 1 ticket
- setting up a movie night adds 2 tickets
- joining a movie night adds 1 ticket

### CODE REVIEW

File: backend/movies/models.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Movie(models.Model):
 
 class User(AbstractUser):
     avatar = models.ImageField(upload_to='avatars/', null=True, blank=True)
+    tickets = models.IntegerField(default=1)
```
Comment (LissaGreense) [inline backend/movies/models.py:17]: image
What do you think about those types for this Field?

File: backend/movies/views.py [hunk 2]
```diff
@@ -49,11 +49,20 @@ def get(self, request, format=None):
 
     def post(self, request, format=None):
         movie_from_body = json.loads(request.body)
+
+        print(movie_from_body)
```
Comment (LissaGreense) [inline backend/movies/views.py:53]: remove

**Review by LissaGreense — APPROVED** [review]
Wonderful work! Thank you again! :smile_cat:

File: backend/movies/models.py (hunk 1 above)
Comment (adlbrtbsr) [reply backend/movies/models.py]: good field

File: backend/movies/views.py (hunk 2 above)
Comment (adlbrtbsr) [reply backend/movies/views.py]: removed

**Pull request merged**